com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -333,6 +333,9 @@ public IReadOnlyList<ulong> ConnectedClientsIds
         /// </summary>
         public bool IsConnectedClient { get; internal set; }
 
+
```

Comment:
Really, we should have a few states for the NetworkManager to provide the NetworkManager's current state.
This is a minimal fix in order to determine if we are in the middle of a shutdown.